code/datums/status_effects/debuffs/debuffs.dm

```diff
@@ -1,3 +1,6 @@
+#define HEALING_SLEEP_DEFAULT 0.2
+#define HEALING_SLEEP_ORGAN_MULTIPLIER 5
```

Comment:
Please autodoc any defines you add (or move) :)

Reply:
Applied with https://github.com/tgstation/tgstation/pull/76679/commits/5c63188a4beb6d12f01bc2919f5a998b7ca4a221